tools/deploy

```diff
@@ -38,28 +38,72 @@ upload_chrome_prod_ext() {
     --refresh-token $CHROME_WEBSTORE_REFRESH_TOKEN
 }
 
-sign_firefox_ext() {
+# Upload a Firefox extension to addons.mozilla.org for automated validation and
+# signing. Write log output to `ext_source_dir/log.txt`.
+#
+# Arguments:
+#   ext_id - ID of the Firefox extension
+#   ext_source_dir - Directory containing source files for the extension.
+# Returns:
+#   0 if extension was successfully signed and downloaded to `ext_source_dir`.
+run_webext_sign() {
+  ext_id=$1
+  ext_source_dir=$2
+
+  # Make pipeline below fail if `webext` fails.
+  set -o pipefail
```

Comment:
Without this, `|`-separated pipeline's exit status is the exit status of the final command (ie. the `grep` calls below).